src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import Button from './Button'
+
+describe('Button', () => {
+  const  { container } = render(<Button>Найти</Button>)
```

Comment:
Не забывай форматировать :-)
```suggestion
  const { container } = render(<Button>Найти</Button>)
```